app/etc/pulp/server.yaml

```diff
@@ -37,10 +37,7 @@
 #
 # logging:
 #   loggers:
-#     django:
-#       level: INFO
-#     pulp.app:
-#       level: INFO
+#     '':
```

Comment:
When I uncomment the current section and push it through yamllint it says it's valid , but it also mutated it some:

```
--- 
logging: 
  loggers: 
    ? ""
    : 
      handlers: 
        - stream
```